backend/shared/shared/common/utils.py

```diff
@@ -137,3 +137,33 @@ def social_security_number_birthdate(social_security_number) -> date:
     century_char = compacted_social_security_number[6]
     century = {"+": 1800, "-": 1900, "A": 2000}[century_char]
     return date(year=century + year_mod_100, month=month, day=day)
+
+
+def create_finnish_social_security_number(
```

Comment:
Perhaps this util function could be under tests folder if it's only used in tests?

Reply:
Taken into account